requirements.in

```diff
@@ -1,8 +1,8 @@
-boltons==24.0.0
+# boltons==24.0.0
```

Comment:
Why are there lines commented out in this file?

Reply:
I meant to remove them. These are unused dependencies. I'm happy to split this out into a separate PR.

Reply:
I've moved this change to #2962.